### PR TITLE
Refactor models

### PR DESCRIPTION
**Pull request by GregJMitchell**
This Branch Does:
- Refactors models and methods

### CODE REVIEW

File: app/models/item.rb [hunk 1]
```diff
@@ -43,10 +42,10 @@ def order_quantity(order_id)
   end
 
   def status(order_id)
-    self.item_orders.where('order_id =?', order_id).pluck(:status)[0]
+    self.item_orders.find_by('order_id =?', order_id).status
```
Comment (abcdefghijohn) [inline app/models/item.rb:45]: Love the use of find_by here to get a single value

**Review by abcdefghijohn — APPROVED** [review]
Looks great!

File: app/models/merchant.rb [hunk 2]
```diff
@@ -30,13 +33,14 @@ def distinct_cities
...
   def disable_merchant
-    self.update(enabled?: false)
-    self.items.update(active?: false)
+    update(enabled?: false)
+    items.update(active?: false)
   end
 
   def enable_merchant
-    self.update(enabled?: true)
-    self.items.update(active?: true)
```
Comment (jakeheft) [inline app/models/merchant.rb:40]: I like taking self out here. Have we done that everywhere?

File: app/models/order.rb [hunk 3]
```diff
@@ -23,12 +23,6 @@ def return_items
...
   end
 
-  def order_status
-    if item_orders.where(status: "Pending") == []
-      update(status: "Packaged")
-    end
-  end
-
   def item_qty(merchant_id)
     self.item_orders.where('merchant_id = ?', merchant_id).sum(:quantity)
```
Comment (jakeheft) [inline app/models/order.rb:27]: Remove self?

**Review by jakeheft — APPROVED** [review]
Nice refactoring. Do we want to stick to

**Pull request merged**